### PR TITLE
Sync changes with the 320

### PR DESCRIPTION
**Pull request by thiagodefreitas**
With velocity mode

### CODE REVIEW

File: ipa_canopen_master/CMakeLists.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ find_package(catkin REQUIRED COMPONENTS
 )
 
 ## System dependencies are found with CMake's conventions
-# find_package(Boost REQUIRED COMPONENTS system)
+find_package(Boost REQUIRED COMPONENTS chrono)
```
Comment (floweisshardt) [inline ipa_canopen_master/CMakeLists.txt:12]: do we need to add boost as a dependency to package.xml?

File: ipa_canopen_test/CMakeLists.txt [hunk 2]
```diff
@@ -38,10 +38,10 @@ catkin_package(
 include_directories(${Boost_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
 
 add_executable(ipa_canopen_test src/test.cpp)
-target_link_libraries(ipa_canopen_test ${catkin_LIBRARIES})
+target_link_libraries(ipa_canopen_test ${catkin_LIBRARIES} ${Boost_LIBRARIES})
```
Comment (floweisshardt) [inline ipa_canopen_test/CMakeLists.txt]: do we need to add boost as a dependency to package.xml?

**Comment by mathias-luedtke** [comment]
I have added boost as dependency for ipa_canopen_master and removed the explicit boost library references from ipa_canopen_test in my fork.

**Comment by floweisshardt** [comment]
@ipa-mdl: can you send a PR for that?

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**